cardano-api/src/Cardano/Api/InMode.hs

```diff
@@ -6,12 +6,17 @@
 -- | Transactions in the context of a consensus mode, and other types used in
 -- the transaction submission protocol.
 --
-module Cardano.Api.TxInMode (
+module Cardano.Api.InMode (
```

Comment:
What's the motivation for the module rename?

Maybe the comment for the module needs adjustment too?